.storybook/helpers/getArgTypes.ts

```diff
@@ -40,8 +40,32 @@ export function getArgTypes(component, options = { variables: {}}) {
 
     return name
   }
+  const getDefaultValue = (value: string) => {
+    if(value.length < 1) {
+      return value
+    }
+    if(value === 'false') {
+      return false
+    }
+    if(value === 'true') {
```

Comment:
`if (typeof value === 'boolean) return value;`

Reply:
value: string 🙈 required this mapping